src/collections/Grid/GridRow.js

```diff
@@ -10,14 +10,14 @@ export default class GridRow extends Component {
 
   static _meta = {
     library: META.library.semanticUI,
-    name: 'Row',
+    name: 'GridRow',
     type: META.type.collection,
     parent: 'Grid',
   };
 
   render() {
     const classes = classNames(
-      'sd-row',
+      'sd-grid-row',
```

Comment:
Just looking at these classes, it would be cool to have these be automated based off of `_meta.name`.